proseco/guide.py

```diff
@@ -183,19 +183,34 @@ def search_stage(self, stage):
 
         n_sigma = stage['SigErrMultiplier']
 
+        # For the reasonable mag check, override the SigErrMultiplier to be at least 1.
+        # It may go to 0 in later stages for the spoiler checks.
+        mag_err_mult = max(n_sigma, 1)
+
+        # And for bright stars, use a local mag_err that is lower bounded at 0.1
+        # for the mag selection.
+        mag_err = cand_guides['mag_err']
+        bright = cand_guides['mag'] < 7.0
+        mag_err[bright] = mag_err[bright].clip(0.1)
+
         # Check reasonable mag
         bright_lim = stage['MagLimit'][0]
         faint_lim = stage['MagLimit'][1]
-        bad_mag = (((cand_guides['mag'] - n_sigma * cand_guides['mag_err']) < bright_lim) |
-                   ((cand_guides['mag'] + n_sigma * cand_guides['mag_err']) > faint_lim))
+        # Confirm that the star mag is not outside the limits when padded by error
+        # and explicitly confirm that the star is not within 2 * mag_err of the hard
+        # bright limit.
+        bad_mag = (
+            ((cand_guides['mag'] - mag_err_mult * mag_err) < bright_lim) |
+            ((cand_guides['mag'] + mag_err_mult * mag_err) > faint_lim) |
+            ((cand_guides['mag'] - 2 * mag_err) < 5.8))
```

Comment:
And here I've coded to the aca_preview test a bit.  This selection code will now never select a star that would hit the critical warning we have now in aca_preview.  

I could get rid of the possibility of the aca_preview orange warning altogether if we set this to 3 * mag_err, but even if we are just talking about 0.1 mags I think that if we get to stage 5 it might be better to bring in the 6.0 mag star with a warning and change that aca_preview warning to a caution... since the selection has been improved so the 6.0 mag star won't be selected unless it has small mag error.

Reply:
For a bright star with mag_err of 0.5, that means this PR selection is

Stage 1: limit 6.2, multiplier 3, brightest possible star 7.7
Stage 2: limit 6.2, multiplier 2, brightest possible star 7.2
Stage 3: limit 6.1, multiplier 1, brightest possible star 6.8 (2 * mag_err from 5.8)

So the question is for that large error star with mag 6.8, do we want to select it in stage 3 with a caution from aca_preview, select it later (fiddle with stages some more), or not select at all?